config.json

```diff
@@ -1599,6 +1602,7 @@
                 "pagination.itemsPerPage": {
                     "type": "inputText",
                     "required": false,
+                    "deprecated": true,
```

Comment:
@stentrop Kam dieser Wert nicht erst in Verbindung mit dem ItemsPerPage-Widget rin?

Reply:
ja kam er, sollte also nicht deprecated werden

Reply:
Setting wurde dedeprecated.

Reply:
Setting wurde dedeprecated.